src/common/mailing.ts

```diff
@@ -90,10 +91,18 @@ interface EmailContact {
 }
 
 const profileToContact = (profile: User, contactId: string) => {
+  const genericInviteURL = getInviteLink({
+    referralOrigin: 'generic',
+    userId: profile.id,
+  });
+
   const contact: EmailContact = {
     id: contactId,
     email: profile.email,
-    custom_fields: { e1_T: profile.id },
+    custom_fields: {
+      e1_T: profile.id,
+      e2_T: genericInviteURL.toString(),
```

Comment:
Just to confirm, have you created it on sendgrid already? (the field I mean)

Reply:
Yep and run the initial CSV already

Reply:
Perfect!